ppp_datamodel/nodes/triple.py

```diff
@@ -16,3 +17,22 @@ def _check_attributes(self, attributes):
         if not all(isinstance(x, AbstractNode) for x in attributes.values()):
             raise TypeError('One of Triple\'s constructor argument '
                             'is not an AbstractNode instance.')
+
+    def predicate_among(self, L):
+        # Be quite permissive about what we accept as L and convert it to
+        # a (frozen)set.
+        if isinstance(L, AbstractNode):
+            L = frozenset({L})
+        elif not isinstance(L, (set, frozenset)):
```

Comment:
No, but if I remove it I have to add an `isinstance(L, (set, frozenset))` clause to the two next conditions.
(or make a useless copy of the set)